### ISSUE TITLE
rpmbuild issue and question

### ISSUE THREAD

**Issue by pjwelsh** [ossc-db/pg_bulkload#144]
At least the SPECS/pg_bulkload-pg15.spec file needs the addition of a "BuildRequires" that has the addition of "ccache":
BuildRequires: ccache, postgresql15-devel, postgresql15

How are you using "rpmbuild" to build the different rpm version of pg_bulkload?
Normally I would just use "rpmbuild -tb some.tar.gz", but since there are several of the specs in the pg_bulkload-3.1.20.tar.gz, rpmbuild is unhappy:
rpmbuild -tb pg_bulkload-3.1.20.tar.gz
error: Found more than one spec file in pg_bulkload-3.1.20.tar.gz
Basically, what is the magic foo params/exports you are using to build the rpms? I've not been able to google myself to a solution that does not involve either modifying the tar file or extracting it.

Thanks
